index.html

```diff
@@ -14,6 +14,9 @@
         <link href="@css/standard_colors_classes.css" rel="stylesheet" type="text/css">
 
         <!--link href="@fonts/css/all.css	" rel="stylesheet" type="text/css"-->
```

Comment:
- [ ] It is good practice to remove all commented code to make your work clean 😄